jax_cosmo/background.py

```diff
@@ -189,6 +191,38 @@ def Omega_de_a(cosmo, a):
   """
   return cosmo.Omega_de*np.power(a, f_de(cosmo, a))/Esqr(cosmo, a)
 
+
+def gamma_growth_rate(cosmo, a):
```

Comment:
hummmmmm.... I worry about adding a second function for the growth rate. Because how the growth is computed will affect things like power spectrum computations, and normalization for instance. So we want to make sure that all computations are consistent.
I'll elaborate more in the review